instrumentation/opentelemetry-instrumentation-sqlalchemy/src/opentelemetry/instrumentation/sqlalchemy/engine.py

```diff
@@ -95,6 +98,8 @@ def _wrap_connect_internal(func, module, args, kwargs):
 
 
 class EngineTracer:
+    _remove_event_listener_params = []
```

Comment:
suggestion: maybe drop the 'remove'
like, call it `event_listener_params`, or even `event_listeners`

Reply:
but the params are for the remove function
I called the list like this so it will be clear that I keep the params only for remove

Reply:
i agree with the `event_listeners` name convention because that what you store in this list

Reply:
Not exactly... I don't store the event listeners params, its without args and kw for exmaple its without(retval=True)
I store the remove event listernes params